tests/unit_tests/main_test.js

```diff
@@ -40,6 +40,114 @@ context("HintCoordinator", () => {
   });
 });
 
+context("Next zoom level", () => {
+  // NOTE: All these tests use the Chrome zoom levels, which are the default!
+  should("Zoom in 0 times", async () => {
```

Comment:
The first 5 tests are for basic zooming with counts in the normal "happy" cases.